src/api/libcellml/units.h

```diff
@@ -402,11 +402,11 @@ class LIBCELLML_EXPORT Units: public NamedEntity, public ImportedEntity
 
     /**
      * @brief Return a bool which indicates if the units are loosely comparable.
-	 * 
-	 * Two units are loosely comparable if they share the same units ONLY (kg,s,m,...),
-	 * this means we are uninterested in unit multipliers. 
-	 *
-	 * @param1 units1 The first units to compare.
+	   * 
+	   * Two units are loosely comparable if they share the same units ONLY (kg,s,m,...),
+	   * this means we are uninterested in unit multipliers. 
+	   *
+	   * @param1 units1 The first units to compare.
```

Comment:
Please, configure your editor properly. No tabs, only spaces!

Reply:
`Tools -> Options -> Text Editor -> [Language] -> Tabs` then change to insert spaces? I did that prior but I'll re-check to make sure it hasn't magically decided to revert back.

Reply:
I don't know which editor you are using (Visual Studio?), but personally I am using [Qt Creator](https://www.qt.io/development-tools) and it's all working fine for me there.

Reply:
Yeah, I'm using Visual Studio. To be extra safe I just went through and re-indented the `units.h` docstrings using spaces only, so hopefully that resolves the issue.